### ISSUE TITLE
Update names, description & representing images of applets

### ISSUE THREAD

**Issue by Jacob-Vlijm** [UbuntuBudgie/budgie-welcome#265]
Some of the applets have outdated images on Welcome (like shuffer). Other applets would benefit a name change (like Wallstreet), to make clear what it represents. Many applet descriptions would benefit using certain _keywords_, to make sure people's eyes would catch it if just scanning the applet's description. 
This is especially more important now the number of applets has increased over time.

**Comment by fossfreedom** [comment]
Fair enough - this also would benefit with keyword search https://github.com/UbuntuBudgie/budgie-welcome/issues/217
